zypp/misc/LoadTestcase.h

```diff
@@ -0,0 +1,130 @@
+/*---------------------------------------------------------------------\
+|                          ____ _   __ __ ___                          |
+|                         |__  / \ / / . \ . \                         |
+|                           / / \ V /|  _/  _/                         |
+|                          / /__ | | | | | |                           |
+|                         /_____||_| |_| |_|                           |
+|                                                                      |
+\---------------------------------------------------------------------*/
+/** \file	zypp/misc/LoadTestcase.h
+ *
+*/
+#ifndef ZYPP_MISC_LOADTESTCASE_H
+#define ZYPP_MISC_LOADTESTCASE_H
+
+#include <zypp/Arch.h>
+#include <zypp/Locale.h>
+#include <zypp/Pathname.h>
+#include <zypp/ResolverFocus.h>
+#include <zypp/Url.h>
+#include <zypp/base/PtrTypes.h>
+#include <zypp/base/SetTracker.h>
+#include <zypp/base/NonCopyable.h>
+#include <zypp/sat/Queue.h>
+#include <zypp/target/modalias/Modalias.h>
+
+#include <optional>
+#include <memory>
+
+namespace zypp::misc {
```

Comment:
I would not use the `misc` namespace here. In case we extend the DefaultLoadSystem stuff, it may collide. Maybe zypp::misc::testcase or so.

Reply:
Done